minimal/Dockerfile.fs

```diff
@@ -0,0 +1,21 @@
+FROM busybox:stable-glibc
+
+COPY --chmod=755 files.tmp/lib/* /lib/x86_64-linux-gnu/
+COPY --chmod=755 files.tmp/bin/* /bin/
+
+RUN mkdir -p /home/ubuntu/software/jasminegraph/conf
+RUN mkdir -p /home/ubuntu/software/jasminegraph/k8s
+RUN mkdir -p /home/ubuntu/software/jasminegraph/metadb
+RUN mkdir -p /home/ubuntu/software/jasminegraph/performancedb
+RUN mkdir -p /home/ubuntu/software/jasminegraph/src/metadb
```

Comment:
Why do we use `src` folder here? the `src` folder is not supposed to have any runtime data.

Reply:
Moved the DDL files to a separate directory outside `src`

Reply:
Ok